src/icons/collections/__tests__/IconCollectionsAdd-test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import IconCollectionsAdd from '../IconCollectionsAdd';
+
+describe('icons/collections/IconCollectionsAdd', () => {
```

Comment:
I'm not sure these unit tests are totally necessary. I can be convinced to remove them.

Reply:
It looks like other Icons in this project use snapshots for testing icons. I think either way is ok.

Reply:
I'll add snapshots. That's a good idea.